cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -207,8 +207,10 @@ class LeadImpl extends ServerImpl with Lead with Logging {
 
   private[snappydata] def internalStop(shutdownCredentials: Properties): Unit = {
     bootProperties.clear()
-    SnappyContext.stop()
     // TODO: [soubhik] find a way to stop jobserver.
+		if (SnappyContext.globalSparkContext != null && !SnappyContext.globalSparkContext.isStopped) {
+			SnappyContext.globalSparkContext.stop
+		}
```

Comment:
globalSparkContext is a method, so better to get it into a val

please also correct the indentation